### PR TITLE
Release/1.3.5

### PR DESCRIPTION
**Pull request by justinengland**
Prospective PR to main, to get installers for 1.3.5 beta

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,9 +8,28 @@ for setuptools_scm/PEP 440 reasons.
...
 
-## 1.3.4 Chia blockchain 2022-4-19
+## 1.3.5 Chia blockchain 2022-5-11
+
+### Added
+
+Added Support for Python 3.10
+Added mempool rules making the op_div mempool rule stricter to match the forthcoming soft-fork
+Fixed cat lineage store when creating new cat wallets
+Added a retry interval for pool info updates
```
Comment (xdustinface) [inline CHANGELOG.md]:
```suggestion
```

Same here for https://github.com/Chia-Network/chia-blockchain/pull/11076

**Review by xdustinface — DISMISSED** [review]
Seems like all those are not in there?

**Review by hoffmang9 — APPROVED** [review]
lgtm

**Comment by wallentx** [comment]
> Seems like all those are not in there?

These were all redone

**Review by cmmarslender — APPROVED** [review]

**Pull request merged**